servers/plain/src/main/docker/Dockerfile

```diff
@@ -10,12 +10,18 @@ RUN yum install -y unzip wget && yum -q clean all
 
 ARG CACHEBUST=1
 
+## PostgreSQL
+ENV psql_module_dir=$JBOSS_HOME/modules/org/postgresql/main/
+ENV psql_connector_jar=postgresql-jdbc.jar
+RUN mkdir -p ${psql_module_dir}
+RUN wget -O ${psql_connector_jar} http://search.maven.org/remotecontent\?filepath\=org/postgresql/postgresql/42.2.2/postgresql-42.2.2.jar
+RUN mv ${psql_connector_jar} ${psql_module_dir}
+
+ADD ${postgresConfigFile} ${psql_module_dir}/module.xml
+
 # WildFly configuration file ready for JMS and HSQL
 ADD ${configFile} $JBOSS_HOME/standalone/configuration/standalone.xml
 
-# Switch to the working dir /opt/jboss/wildfly
-WORKDIR /opt/jboss/wildfly
-
 # Run everything below as aerogear user
```

Comment:
No part of this change, but I believe this should read `Run everything below as 'jboss' user`

Reply:
Agreed. A comment probably isn't needed for this even (a lot of the comments here could probably be removed)!